app/views/projects/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2> Project name: <%=@project.name %></h2>
+  <p>Material: <%=@project.material %></p>
+  <p>Challenge Theme: <%= @theme %></p>
```

Comment:
See my comment in your controller. I would just call `@project.theme` here and not create an instance variable in your controller.